src/app/drive/services/download.service/downloadFolder/downloadSharedFolderUsingFileSystemAPI.ts

```diff
@@ -28,15 +28,16 @@ export async function downloadSharedFolderUsingFileSystemAPI(
     progressCallback: (downloadedBytes: number) => void;
   },
 ): Promise<void> {
+  const { translate } = useTranslationContext();
```

Comment:
Be careful, hooks not will be work outside of react components. Review all functions that has added this hook and change it.